imap_processing/spice/geometry.py

```diff
@@ -320,3 +337,42 @@ def get_rotation_matrix(
         otypes=[np.float64],
     )
     return vec_pxform(from_frame.name, to_frame.name, et)
+
+
+def instrument_pointing(
+    et: Union[float, npt.NDArray],
+    instrument: SpiceFrame,
+    to_frame: SpiceFrame,
+    cartesian: bool = False,
+) -> npt.NDArray:
+    """
+    Compute the instrument pointing at the specified times.
+
+    By default, the coordinates returned are Latitude/Longitude coordinates in
+    the reference frame `to_frame`. Cartesian coordinates can be returned if
+    desired by setting `cartesian=True`.
+
+    Parameters
+    ----------
+    et : float or npt.NDArray
+        Ephemeris time(s) to at which to compute instrument pointing.
+    instrument : SpiceFrame
+        Instrument reference frame to compute the pointing for.
+    to_frame : SpiceFrame
+        Reference frame in which the pointing is to be expressed.
+    cartesian : bool, optional
+        If set to True, the pointing is returned in Cartesian coordinates.
+
+    Returns
+    -------
+    pointing : npt.NDArray
+        The instrument pointing at the specified times.
+    """
+    pointing = frame_transform(
+        et, BORESIGHT_LOOKUP[instrument.name], instrument, to_frame
+    )
+    if cartesian:
+        return pointing
+    if isinstance(et, typing.Collection):
+        return np.array([spice.reclat(vec)[1:] for vec in pointing])
+    return np.array(spice.reclat(pointing)[1:])
```

Comment:
`spice.reclat()` returns radians. These should probably be converted to degrees.